keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,742 @@
+        "message": "HTTP authentication with multiple credentials detected. Click on the extension icon to choose the correct one.",
+        "description": "Notification when HTTP Authentication has multiple credentials detected."
+    },
+    "errorMessageUnknown": {
+        "message": "Unknown error.",
+        "description": "Unknown error."
+    },
+    "errorMessageDatabaseNotOpened": {
+        "message": "Database not opened.",
+        "description": "Database not opened."
+    },
+    "errorMessageDatabaseHash": {
+        "message": "Database hash not received.",
+        "description": "Database hash not received."
+    },
+    "errorMessageClientPublicKey": {
+        "message": "Client public key not received.",
+        "description": "Client public key not received."
+    },
+    "errorMessageDecrypt": {
+        "message": "Cannot decrypt message.",
+        "description": "Cannot decrypt message."
+    },
+    "errorMessageTimeout": {
+        "message": "Timeout or not connected to KeePassXC.",
+        "description": "Timeout or not connected to KeePassXC."
+    },
+    "errorMessageCancelled": {
+        "message": "Action cancelled or denied.",
+        "description": "Action cancelled or denied."
+    },
+    "errorMessageEncrypt": {
+        "message": "Cannot encrypt message or public key not found. Is native messaging or support for your browser enabled in KeePassXC?",
+        "description": "Cannot encrypt message or public key not found. Is native messaging or support for your browser enabled in KeePassXC?"
+    },
+    "errorMessageAssociate": {
+        "message": "KeePassXC association failed, try again.",
+        "description": "KeePassXC association failed, try again."
+    },
+    "errorMessageKeyChange": {
+        "message": "Key change was not successful.",
+        "description": "Key change was not successful."
+    },
+    "errorMessageEncryptionKey": {
+        "message": "Encryption key is not recognized.",
+        "description": "Encryption key is not recognized."
+    },
+    "errorMessageSavedDatabases": {
+        "message": "No saved databases found.",
+        "description": "No saved databases found."
+    },
+    "errorMessageIncorrectAction": {
+        "message": "Incorrect action.",
+        "description": "Incorrect action."
+    },
+    "errorMessageEmptyMessage": {
+        "message": "Empty message received.",
+        "description": "Empty message received."
+    },
+    "errorMessageNoURL": {
+        "message": "No URL provided.",
+        "description": "No URL provided."
+    },
+    "errorMessageNoLogins": {
+        "message": "No logins found.",
+        "description": "No logins found."
+    },
+    "passwordGeneratorPlaceholder": {
+        "message": "Generated password",
+        "description": "Input field placeholder for generated password."
+    },
+    "passwordGeneratorLabel": {
+        "message": " also fill in the next password-field",
```

Comment:
This looks to me like it's going to be stitched together with another string. Use placeholders instead, English word and clause order isn't the standard in all languages.

Reply:
This is a checkbox label. I think it's best to add the space to the HTML layout and make this sentence to start with uppercase.